packages/documentation/astro.config.mjs

```diff
@@ -77,8 +77,77 @@ export default defineConfig({
                 },
                 {
                   label: 'Deploy to production',
-                  link: 'integration/deploy-to-production'
-                },
+                  collapsed: true,
+                  items: [
+                    {
+                      label: 'Endpoints',
+                      collapsed: true,
+                      items: [
+                        {
+                          label: 'Webhook events',
+                          link: 'integration/webhook-events'
+                        },
+                        {
+                          label: 'Exchange Rates',
+                          link: 'integration/exchange-rates'
+                        },
+                        {
+                          label: 'IDP',
+                          link: 'integration/idp'
+                        }
+                      ]
+                    },
+                    {
+                      label: 'Environment variables',
+                      link: 'integration/environment-variables'
+                    },
+                    {
+                      label: 'Deploying to cloud environment options',
+                      collapsed: true,
+                      items: [
+                        {
+                          label: 'nginx',
+                          link: 'integration/nginx'
+                        },
+                        {
+                          label: 'Helm & K8s',
+                          link: 'integration/helm-k8s'
+                        }
+                      ]
+                    },
+                    {
+                      label: 'Running your instance',
+                      collapsed: true,
+                      items: [
+                        {
+                          label: 'Adding asset',
+                          link: 'integration/adding-asset'
+                        },
+                        {
+                          label: 'Adding peer',
+                          link: 'integration/adding-peer'
+                        },
+                        {
+                          label:
+                            'Creating wallet address (strategies for identity lookup)',
+                          link: 'integration/creating-wallet-address'
+                        },
+                        {
+                          label: 'Managing liquidity',
+                          link: 'integration/managing-liquidity'
```

Comment:
```suggestion
                          link: 'integration/adding-liquidity'
```

Reply:
Similar comment to above.  The Adding Liquidity page has been renamed to Managing Liquidity.